lxd/main_recover.go

```diff
@@ -146,7 +146,7 @@ func (c *cmdRecover) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	fmt.Printf("The recovery process will be scanning the following storage pools:\n")
+	fmt.Print("The recovery process will be scanning the following storage pools:\n")
```

Comment:
Println candidate?

Reply:
I initially went with `Print()` to follow the existing pattern in that file but I've now made them all `Println()`, thanks!